website/docs/ephemeral-resources/outputs.html.markdown

```diff
@@ -0,0 +1,61 @@
+---
+layout: "tfe"
+page_title: "Terraform Enterprise: Ephemeral: tfe_outputs"
+description: |-
+  Get output values from another organization/workspace without writing
+  sensitive data to state.
+---
+
+# Ephemeral: tfe_organization_token
+
+Terraform ephemeral resource for managing a TFE organization token. This
+resource is used to generate a new organization token that is guaranteed not to
+be written to state. Since organization tokens are singleton resources, using this ephemeral resource will replace any existing organization token, including those managed by `tfe_organization_token`.
+
+This ephemeral resource is used to retrieve the state outputs for a given workspace.
+It enables output values in one Terraform configuration to be used in another.
+The retrieved output values are guaranteed not to be written to state.
+
+~> **NOTE:** Ephemeral resources are a new feature and may evolve as we continue to explore their most effective uses. [Learn more](https://developer.hashicorp.com/terraform/language/v1.10.x/resources/ephemeral).
+
+---
+
+
+~> **NOTE:** The `values` attribute is preemptively marked [sensitive](https://developer.hashicorp.com/terraform/language/values/outputs#sensitive-suppressing-values-in-cli-output) and is only populated after a run completes on the associated workspace. Use the `nonsensitive_values` attribute to access the subset of the outputs
+that are known to be non-sensitive.
+
+## Example Usage
+
+Using the `tfe_outputs` ephemeral resource, the outputs `foo` and `bar` can be used as seen below:
+
+In the example below, assume we have outputs defined in a `my-org/my-workspace`:
+
+```hcl
+ephemeral "tfe_outputs" "foo" {
+  organization = "my-org"
+  workspace = "my-workspace"
+}
+
+resource "random_id" "vpc_id" {
+  keepers = {
+    # Generate a new ID any time the value of 'bar' in workspace 'my-org/my-workspace' changes.
+    bar = ephemeral.tfe_outputs.foo.values.bar
+  }
+
+  byte_length = 8
+}
+```
+
+## Argument Reference
+
+The following arguments are supported:
+
+* `organization` - (Required) The name of the organization.
+* `workspace` - (Required) The name of the workspace.
+
+## Attributes Reference
+
+The following attributes are exported:
+
+* `values` - The current output values for the specified workspace.
+* `nonsensitive_values` - The current non-sensitive output values for the specified workspace, this is a subset of all output values.
```

Comment:
Do we need this property on this resource? My hunch is that the restrictions are so high for ephemeral values that it would eclipse sensitive value restrictions. You tell me

Reply:
Hmm, good question, I can see an argument removing it, but also keeping it around. From a data perspective, the information provided by this resource is lossy if we don't include both attributes - the HCPT-defined Sensitive flag is dropped. I could see that maybe being useful to users in certain situations.